pkg/services/rpcsrv/client_test.go

```diff
@@ -2466,6 +2468,43 @@ func TestClient_GetVersion_Hardforks(t *testing.T) {
 	require.InDeltaMapValues(t, expected, v.Protocol.Hardforks, 0)
 }
 
+func TestServer_GetVersion(t *testing.T) {
+	chain, srv, _ := initServerWithInMemoryChain(t)
```

Comment:
For this test you need an empty server without test blocks. Use `getUnitTestChainWithCustomConfig` + `wrapUnitTestChain` instead, ensure that `SaveInvocations` and `KeepOnlyLatestState` are set in the node's config.